src/Teams/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -284,6 +283,31 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("SingleSelectionQuestionOption");
                 });
 
+            modelBuilder.Entity("Teams.Models.QueuedProgram", b =>
+                {
+                    b.Property<long>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnName("Id")
+                        .HasColumnType("bigint")
+                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+                    b.Property<string>("Program")
+                        .HasColumnName("program")
+                        .HasColumnType("nvarchar(max)");
+
+                    b.Property<Guid>("QuestionId")
```

Comment:
Can quiestionId be a foreign key for Question?

Reply:
I found that in order to make it a foreign key, I need to add something like .HasOne(q => q.Question)
            .WithMany(p => p.QueuedPrograms).HasForeignKey... As I understand, this approach requires to add a reference to the list of the queued programs to the ProgramCodeQuestion class. But I think it is not a good idea, because the ProgramCodeQuestion entity shouldn't contain some external answers for it. Maybe I missed something, but how to solve this problem correctly?

Reply:
Could you review this piece of documentation?
https://docs.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key#without-navigation-property

Reply:
Ok, thank you.